web-client/src/presenter/actions/checkDocumentTypeAction.ts

```diff
@@ -0,0 +1,15 @@
+export const checkDocumentTypeAction = ({ path, props }: ActionProps) => {
+  const parentMessageId = props.parentMessageId
+    ? `/${props.parentMessageId}`
+    : '';
+
+  if (props.documentType === 'Miscellaneous') {
```

Comment:
Constant?

Reply:
I'm not sure what the best constant is for this. We have https://github.com/ustaxcourt/ef-cms/blob/f63ddafe019ce47dae1c22631e94c3b591e81a00/shared/src/business/entities/EntityConstants.ts#L1580, but I am unsure this is a practitioner document? It looks like we use this 'Miscellaneous' check in a few places (hence the reason we did not use a constant either), so whatever we decide should be used across these cases:

- https://github.com/ustaxcourt/ef-cms/blob/f63ddafe019ce47dae1c22631e94c3b591e81a00/web-client/src/presenter/utilities/getDocumentEditUrl.ts#L2-L11
- https://github.com/ustaxcourt/ef-cms/blob/f63ddafe019ce47dae1c22631e94c3b591e81a00/shared/src/business/utilities/getFormattedCaseDetail.ts#L260

Reply:
It may have to be a new constant; I don't think these are practitioner documents

Reply:
@akuny @pixiwyn It turns out that this check is actually not working properly: `documentType` is `undefined`, so editing a miscellaneous document upload does not take a user to the appropriate screen (and it didn't do so before our changes, either--this is the current behavior on prod). We suspect there have not been complaints because the user can just delete the draft and re-upload.

After talking with @TomElliottFlexion and @cholly75, we decided to keep the PR as-is (since it is working) and tackle the bug (and add tests!) in one fell swoop as a separate Zenhub issue.

Reply:
Big props for investigating that @Mwindo